src/tabpfn_extensions/hpo/tuned_tabpfn.py

```diff
@@ -373,28 +381,10 @@ def objective(params):
         self.trials_ = trials_obj  # Store the modified trials history
 
         if self.best_model_ is None:
-            warnings.warn(
-                "All optimization trials failed or no valid model was found. Creating default model.",
-                stacklevel=2,
+            raise ValueError(
```

Comment:
could it be an issue in AutoTabPFNClassifier if this happens from time to time but is fine as part of the PHE? (No idea if that's actually a thing)